### ISSUE TITLE
Create a smaller dataset for quick testing

### ISSUE THREAD

**Issue by fmellomascarenhas** [OpenBioLink/OpenBioLink#60]
Hi all,

Fastai created 2 small subsets of imagenet dataset so ideas could be quickly tested. They call it Imagenette and imagewoof: https://github.com/fastai/imagenette

To compare results in a leaderboard, they fix 1) Picture size (equivalent to embedding size in graphs); 2) Number of epochs
![image](https://user-images.githubusercontent.com/23004953/96750947-94f66880-139a-11eb-83b6-2b15bcda59cd.png)

I imagine that if OpenBioLink had a subset with a few thousand nodes/edges, people could test their ideas quickly, in their local machine, which would help the field (and the spread of OpenBioLink).

PS: Thanks for creating and sharing such amazing work.

**Comment by matthias-samwald** [comment]
I guess we have to think about this a bit -- the very essence of biological link prediction with the kind of broad knowledge represented by OpenBioLink is that algorithms need to be able to use the large dataset and draw from potentially subtle patterns encoded in it.

Having said that, I _do_ wonder if we should at least define a smaller test set. The 5% test set contains a large number of triples, and many link prediction evaluation methods are currently painfully slow -- evaluation often takes longer than training a model. So perhaps it would make sense to pre-define a 'quick-test set'? @nomisto

**Comment by fmellomascarenhas** [comment]
"algorithms need to be able to use the large dataset and draw from potentially subtle patterns encoded in it."

I completely agree with you. But I wonder if, for example, since there's a lot of research on cancer, a subset with all nodes related to cancer would be enough to evaluate ideas very quickly (different convolutions, losses, hyperparameter, lr schedule, activations, pooling, normalization, emb size,  etc). But, again, I don't know if this would work. Just an idea. Thanks for taking your time to answer.
